src/components/SimMenu.tsx

```diff
@@ -255,7 +267,7 @@ class SimMenu extends React.PureComponent<Props, State> {
           disabled={false}
         >
           <ItemIcon icon={faStop} />
-          Stop
+          {LocalizedString.lookup(tr('Stop', 'Terminate program execution'), locale)}
```

Comment:
Doesn't get translated, even though the translation exists in the PO and JSON files. Seems to be a pattern where none of the `tr(str, str)` (with the second param) work properly